src/main/java/org/opensearch/neuralsearch/query/HybridQueryBuilder.java

```diff
@@ -189,6 +189,13 @@ public static HybridQueryBuilder fromXContent(XContentParser parser) throws IOEx
             } else {
                 if (AbstractQueryBuilder.BOOST_FIELD.match(currentFieldName, parser.getDeprecationHandler())) {
                     boost = parser.floatValue();
+                    // regular boost functionality is not supported, user should use score normalization methods to manipulate with scores
+                    if (boost != DEFAULT_BOOST) {
+                        log.error(
+                            String.format(Locale.ROOT, "[%s] query does not support provided value %.4f for [%s]", NAME, boost, BOOST_FIELD)
```

Comment:
Can we just add the boost value in exception message itself and remove this logging?
```suggestion
                            "[{}] query does not support provided value {} for [{}]", NAME, boost, BOOST_FIELD
```

Reply:
ack, as for the more detailed error message in exception, I rather not do it, got recommendation several times from security team not to put anything user provided to the user facing content, say if this exception is propagated to UI something like this can cause execution of malicious script.

Reply:
True. However, we already validated the input is floatValue in this case right?